packages/errors/scripts/extract-errors.ts

```diff
@@ -0,0 +1,189 @@
+import * as fs from 'fs';
```

Comment:
Feel free to ignore, but I think you can also 

```suggestion
import { promises as fs } from 'fs';
```

since you’re always using them as async functions anyway :)

Reply:
This exists? 😲